src/field_theory/ratfunc.lean

```diff
@@ -910,4 +1077,59 @@ end
 
 end eval
 
+section laurent_series
+
+open power_series laurent_series hahn_series
+
+omit hring
+variables {F : Type u} {S : Type v} [field F] [comm_semiring S] [algebra S (polynomial F)]
+  (p q : polynomial F) (f g : ratfunc F)
+
+/-- The coercion `ratfunc F → laurent_series F` as bundled alg hom. -/
+def coe_alg_hom (F : Type u) [field F] : ratfunc F →ₐ[polynomial F] laurent_series F :=
```

Comment:
We want an instance `algebra (ratfunc F) (laurent_series F)`, with  `is_scalar_tower (polynomial K)...`. I am not completely sure it doesn't follow from what you've already done, but can you add it if it is missing?